dnsimple/dnsimple-developer
# Adds search

## 👀 Preview

![search](https://user-images.githubusercontent.com/80610/179024338-c593d70f-31b7-46ad-a527-253ae0c55054.gif)

## QA

Search for top level concepts, or specific things:
- zones
- ddns
- credit card
- ruby

The experience should match the search in the support site.
